src/styles/global.css

```
@@ -557,15 +581,15 @@
     .input {
         width: 100%;
         padding: 0.75rem 1rem;
-        @apply text-base text-text-primary font-medium;
+        @apply text-primary text-base font-medium;
         background: var(--bg-elevated);
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)